packages/esm-form-engine-app/src/form-renderer/form-renderer.component.tsx

```diff
@@ -55,6 +68,70 @@ const FormRenderer: React.FC<FormRendererProps> = ({
     [promptBeforeClosing],
   );
 
+  const patientDetails = useMemo(() => {
+    const getGender = (gender: string): string => {
+      switch (gender) {
+        case 'male':
+          return t('male', 'Male');
+        case 'female':
+          return t('female', 'Female');
+        case 'other':
+          return t('other', 'Other');
+        case 'unknown':
+          return t('unknown', 'Unknown');
+        default:
+          return gender;
+      }
+    };
+
+    const identifiers =
+      patient?.patient?.identifier?.filter(
+        (identifier) => !excludePatientIdentifierCodeTypes?.uuids.includes(identifier.type.coding[0].code),
+      ) ?? [];
+
+    return {
+      name: patient?.patient ? displayName(patient?.patient) : '',
+      age: age(patient?.patient?.birthDate),
+      gender: getGender(patient?.patient?.gender),
+      location: patient?.patient?.address?.[0].city,
+      identifiers: identifiers?.length ? identifiers.map(({ value, type }) => value) : [],
+    };
+  }, [patient, t, excludePatientIdentifierCodeTypes?.uuids]);
+
+  const headerTitle = t('form', 'form');
```

Comment:
Instead of having the word `form`, wouldn't the form name itself be more suitable as the header? e.g :- 'Adult HIV Return Visit Form'

Reply:
Yah, nice catch @NethmiRodrigo